### PR TITLE
Unified/saas offer

### PR DESCRIPTION
**Pull request by KurtSchenk**

### CODE REVIEW

File: deployment/Deploy.ps1 [hunk 1]
```diff
@@ -347,10 +347,13 @@ Write-host "      ➡️ Execute SQL schema/data script"
 Invoke-Sqlcmd -InputFile ./script.sql -ServerInstance $ServerUri -database $SQLDatabaseName -Username $SQLAdminLogin -Password $SQLAdminLoginPassword 
 
 Write-host "   🔵 Deploy Code to Admin Portal"
-az webapp deploy --resource-group $ResourceGroupForDeployment --name $WebAppNameAdmin --src-path "../Publish/AdminSite.zip" --type zip --output $azCliOutput
+#az webapp deploy --resource-group $ResourceGroupForDeployment --name $WebAppNameAdmin --src-path "../Publish/AdminSite.zip" --type zip --output $azCliOutput
+$AccessToken = (Get-AzAccessToken).Token
+& "curl" -X POST -H "Authorization: Bearer $AccessToken" -T "../Publish/AdminSite.zip" https://$WebAppNameAdmin.scm.azurewebsites.net/api/publish?type=zip -v
```
Comment (alexdrenea) [inline deployment/Deploy.ps1:353]: What is the benefit of this method vs the az cli utilities?

**Comment by alexdrenea** [comment]
I might be missing some context, what is the install button supposed to do?

**Comment by santhoshb-msft** [comment]
@KurtSchenk any thoughts on the above comments ?

**Comment by santhoshb-msft** [comment]
@KurtSchenk  - closing this for now. please re-open when you can help us with above comments?

**Pull request closed without merging**